pkg/ptp/ptp.go

```diff
@@ -104,6 +104,33 @@ type Torrent struct {
 	RemasterYear  *string `json:"RemasterYear,omitempty"`
 }
 
+// custom unmarshal method for Torrent
+func (t *Torrent) UnmarshalJSON(data []byte) error {
+	type Alias Torrent
+
+	aux := &struct {
+		Id interface{} `json:"Id"`
+		*Alias
+	}{
+		Alias: (*Alias)(t),
+	}
+
+	if err := json.Unmarshal(data, &aux); err != nil {
+		return err
+	}
+
+	switch id := aux.Id.(type) {
+	case float64:
+		t.Id = fmt.Sprintf("%.0f", id)
```

Comment:
Shouldn't this test as int64 instead of float64?

Reply:
Oops. Yeah definitely. Will fix!

Reply:
Correction: The numbers are parsed as floats when marshalled so it should stay float